packages/core/package.json

```diff
@@ -25,6 +25,7 @@
     "ajv": "^6.5.3",
     "body-parser": "^1.17.2",
     "cookie": "^0.4.0",
+    "cdaemon": "^0.0.15",
```

Comment:
I believe there are a few issues with the dependency:
- I don't believe we want to introduce a dependency on the project which is an untested and potentially unmaintained. I see that it is from a personal repository.
- The current license (`the unlicense`) is incompatible with this project and is not an Eclipse Foundation [approved license](https://www.eclipse.org/legal/licenses.php#approved).

Reply:
Thanks for the review @vince-fugnitto, this is my first contribution and I appreciate this.

Should I expand BackendApplicationContribution then by adding an onListen() method that I can implement in an extension (and daemonize there)?

And regarding the license, I can relicense.

Reply:
> Should I expand BackendApplicationContribution then by adding an onListen() method that I can implement in an extension (and daemonize there)?

I am sure this change could be accepted way earlier than the current changeset.